### PR TITLE
About Page

### PR DESCRIPTION
**Pull request by kitcath**
Finished a rough draft of the about page

### CODE REVIEW

File: content/pages/about/index.md [hunk 1]
```diff
@@ -2,17 +2,8 @@
 title: 'About'
 date: 2018-12-06T09:29:16+10:00
 layout: 'aboutlayout'
-heroHeading: 'About Us'
-heroSubHeading: "We've been supporting students since 2010. Join us as we celebrate 10 years of learning!"
+heroHeading: 'Supporting Students Since 2010'
```
Comment (kellydanma) [inline content/pages/about/index.md]: Can we remove the capitalization here? 

Eg. `Supporting students since 2010`

Small nitpick I know, it's just to maintain consistency across the website. Capitalization should be used for acronyms (eg. Frequently Asked Questions because the acronym is _FAQ_), but not titles.

Comment (kellydanma) [reply content/pages/about/index.md]: Actually nvm, I see why you capitalized it here. 

It's a bit long, can we use something else? The 10 years is implied by the text right underneath. Perhaps:

- `Supporting Students`
- `Supporting Learners`
- etc...

image

File: themes/hugo-hero-theme/layouts/pages/aboutlayout.html [hunk 2]
```diff
@@ -4,9 +4,77 @@
...
+   
+  </style>
+</head>
+
+<body>
+  <br>
+  <br>
+
+  <div class="welcome">
+  <h1 class="title">Our Mission </h1>
```
Comment (kellydanma) [inline themes/hugo-hero-theme/layouts/pages/aboutlayout.html]: The title is off centre - can we just exclude it?

image

File: content/pages/about/index.md (hunk 1 above)
Comment (kellydanma) [reply content/pages/about/index.md]: N.B. It's okay if it doesn't fit into 1 line on mobile, but it would be nice to fit it into one line on desktop.

File: data/features.json [hunk 3]
```diff
@@ -1,17 +1,19 @@
 [
   {
-    "title": "Free Consultation",
-    "description": "New clients recieve an obligation free consultation.",
-    "image": "features/noun_branding_1885335.svg"
+    "title": "Coding",
+    "description": "Build an awesome project with your team over the course of the weekend (and get feedback and mentoring throughout) ",
```
Comment (kellydanma) [inline data/features.json]: Nitpick here, but this sentence doesn't end with punctuation like the others.

File: themes/hugo-hero-theme/layouts/pages/aboutlayout.html [hunk 4]
```diff
@@ -4,9 +4,77 @@
...
+  
+  <!--
+  <img src= "../../images/about-icon.png" alt="computers" class="img" >
+  -->
+  <p class="text"> 
+                  
+    Since being founded in 2010, in association with McGill’s Electrical,
```
Comment (kellydanma) [inline themes/hugo-hero-theme/layouts/pages/aboutlayout.html:50]: This section is pretty long with the sentences being pretty verbose.

I'm wondering if we can either:
1. Shorten this section by employing shorter sentences
2. Cut it down to 1 paragraph and add 3 more `features` to the about section to incorporate the parts we remove

For example, we could add these points:
- `teamwork`
- `learning`
- `hiring` (this is pretty explicit, but that's _okay_, we want sponsors to hire students)

**Review by kellydanma — CHANGES_REQUESTED** [review]

File: themes/hugo-hero-theme/layouts/pages/aboutlayout.html (hunk 2 above)
Comment (kitcath) [reply themes/hugo-hero-theme/layouts/pages/aboutlayout.html]: I decided to put the title " our Mission" as the header of our page.

**Comment by kitcath** [comment]
I adjusted the page based on your comments. I added more icons and made one paragraph to describe who we are. The page looks a bit empty. Any suggestions to optimize the space we have on the page. 

TL;DR:
1- Made the title fit on one line 
2-Made the paragraph shorter 
3-Added more points in the feature.json file

File: themes/hugo-hero-theme/layouts/pages/aboutlayout.html (hunk 2 above)
Comment (kellydanma) [reply themes/hugo-hero-theme/layouts/pages/aboutlayout.html]: Lit 🎈

**Comment by kellydanma** [comment]
It looks good, I'll merge this into `master` and we can always update parts of the web page incrementally :)

**Review by kellydanma — APPROVED** [review]
